### PR TITLE
Fixes Dribbble description animation

### LINKED ISSUE #675: Dribbble description doesn't slide up with the rest of the content

### PR DESCRIPTION
**Pull request by manuelvicnt**
## :loudspeaker: Type of change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring

## :scroll: Description

Fixes Dribbble description animation.

## :bulb: Motivation and Context

Fixes #675 

For some reason, the transition doesn't like the BindingAdapter `goneUnless`. Even if we modify the adapter to only set the new visibility when it changes, the animation stops working. 

My gut tells me that the view is always GONE because we emit a first ShotUiModel with an empty description and then when the description is loaded, we emit another model with description making the animation for that view to stop working. 

That's why the decision is to set the visibility to GONE in the htmlText BindingAdapter when the description is null.

## :green_heart: How did you test it?
Manually.

## :pencil: Checklist

- [x] I ran `./gradlew spotlessApply` before submitting the PR
- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [x] All tests passing

## :crystal_ball: Next steps
None.

## :camera_flash: Screenshots / GIFs

fix

### CODE REVIEW

File: dribbble/src/main/java/io/plaidapp/dribbble/ui/shot/ShotBindingAdapters.kt [hunk 1]
```diff
@@ -67,7 +68,10 @@ fun bindHtmlText(
     textView: TextView,
```
Comment (nickbutcher) [inline dribbble/src/main/java/io/plaidapp/dribbble/ui/shot/ShotBindingAdapters.kt]: Rename attr & fun to indicate this new behavior? e.g. `htmlTextOrGone` & `bindHtmlTextOrGone`.

File: dribbble/src/main/java/io/plaidapp/dribbble/ui/shot/ShotBindingAdapters.kt [hunk 2]
```diff
@@ -67,7 +68,10 @@ fun bindHtmlText(
...
     htmlText: CharSequence?
 ) {
-    if (htmlText == null) return
+    if (htmlText == null) {
+        textView.visibility = View.GONE
```
Comment (nickbutcher) [inline dribbble/src/main/java/io/plaidapp/dribbble/ui/shot/ShotBindingAdapters.kt]: Import method as elsewhere for readability?

File: dribbble/src/main/java/io/plaidapp/dribbble/ui/shot/ShotBindingAdapters.kt (hunk 1 above)
Comment (manuelvicnt) [reply dribbble/src/main/java/io/plaidapp/dribbble/ui/shot/ShotBindingAdapters.kt]: 👍

File: dribbble/src/main/java/io/plaidapp/dribbble/ui/shot/ShotBindingAdapters.kt (hunk 2 above)
Comment (manuelvicnt) [reply dribbble/src/main/java/io/plaidapp/dribbble/ui/shot/ShotBindingAdapters.kt]: Done! Thanks

**Pull request merged**